ow_gazebo_plugins/src/OWLightControlPlugin/OWLightControlPlugin.h

```diff
@@ -0,0 +1,37 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+#ifndef OWLightControlPlugin_h
+#define OWLightControlPlugin_h
+
+#include <gazebo/common/Plugin.hh>
+#include <ros/ros.h>
+
+
+namespace gazebo {
+  
+class OWLightControlPlugin : public gazebo::VisualPlugin
+{
+public:
+  OWLightControlPlugin();
+  ~OWLightControlPlugin();
```

Comment:
Empty methods in the cpp are unnecessary if you just default them.
```suggestion
  OWLightControlPlugin() = default;
  ~OWLightControlPlugin() = default;
```

Reply:
Changed to `{}` in the header. I believe `default` is intended for trivial types and we don't really need it here.